gensim/models/doc2vec.py

```diff
@@ -529,10 +529,9 @@ def repeat(self, word_count):
 
 class Doc2Vec(Word2Vec):
     """Class for training, using and evaluating neural networks described in http://arxiv.org/pdf/1405.4053v2.pdf"""
-    def __init__(self, documents=None, size=300, alpha=0.025, window=8, min_count=5,
-                 max_vocab_size=None, sample=0, seed=1, workers=1, min_alpha=0.0001,
-                 dm=1, hs=1, negative=0, dbow_words=0, dm_mean=0, dm_concat=0, dm_tag_count=1,
-                 docvecs=None, docvecs_mapfile=None, comment=None, trim_rule=None, **kwargs):
+    def __init__(self, documents=None, window=8, sample=0, dm=1, hs=1, 
+                negative=0, dbow_words=0, dm_mean=0, dm_concat=0, dm_tag_count=1,
+                 docvecs=None, docvecs_mapfile=None, comment=None, **kwargs):
```

Comment:
Since the Word2Vec defaults now include `window=5, sample=1e-3, hs=0, negative=5, cbow_mean=1` some of these values are still overriding those. (`cbow_mean` in Word2Vec is essentially analogous to `dm_mean` in Doc2Vec.) Also, the indention is a little off.